Fix make parallelism issue

- remove pkginclude_HEADERS ( LIBFM_INCLUDES and LIBFM_GTK_INCLUDES
variables are empty)
- if we don't remove it then we will have a race condition between the code 
that tries to symlink ${includedir}/libfm-1.0 to ${includedir}/libfm and the
am autogenerated code from the pkginclude_HEADERS definition which
tries to create pkgincludedir (${includedir}/libfm);
- if pkgincludedir is created before the symlink the symlink will be created
in the ${includedir}/libfm dir and it will have libfm-1.0 as name which is
wrong (we need the ${includedir}/libfm symlink for pcmanfm)

Upstream-Status: Pending
Signed-off-by: Constantin Musca <constantinx.musca@intel.com>

Index: libfm-1.1.0/src/Makefile.am
===================================================================
--- libfm-1.1.0.orig/src/Makefile.am
+++ libfm-1.1.0/src/Makefile.am
@@ -211,11 +211,6 @@ libfmgtkinclude_HEADERS = \
 	gtk/fm-gtk-marshal.h \
 	$(NULL)
 
-pkginclude_HEADERS = \
-	$(LIBFM_INCLUDES) \
-	$(LIBFM_GTK_INCLUDES) \
-	$(NULL)
-
 EXTRA_LTLIBRARIES = libfm-gtk.la libfm-gtk3.la
 
 lib_LTLIBRARIES = libfm.la @LIBFM_GTK_LTLIBRARIES@
